libraries/stdlib/stdlib_ng.mtlx

```diff
@@ -1026,6 +1026,141 @@
     </mix>
     <output name="out" type="vector4" nodename="N_mix_vector4" />
   </nodegraph>
+  
+  <!--
+    <unifiednoise2d>
+    Combined 2d noises for artists.
+  -->
+  <nodegraph name="NG_unifiednoise2d_float" nodedef="ND_unifiednoise2d_float">
+    <range name="N_range" type="float">
```

Comment:
@kwokcb  Sorry for the delay following up; aren't we supposed to split the nodedef and nodegraphs into separate files? I thought that's the pattern followed by other nodegraph-defined nodes; but I'd be happy to merge them in one place (FWIW, I also followed this practice for the other two nodes, so if you and @jstone-lucasfilm are okay moving the nodedefs into the nodegraph document, I'll do it for all 3).

Reply:
Sorry for any confusion. What I meant was to reflect the inputs on the nodedef into the nodegraph. Not to change where they reside.

It will fit the 1.39 change to have nodedefs referenced via an implementaion easier, and 
It also makes the current  implementaion logic do less work when evaluating.

Reply:
Ah! Okay I think I understand, thanks for the explanation @kwokcb 

Do I need to include docs on the inputs in both places, or just the nodedef document?

Reply:
Documentation in the nodedef element is fine, and the documentation above looks good to me!